dev-tools.sh

```diff
@@ -223,6 +223,29 @@ dev-xdebug-init() {
 	echo
 }
 
+#gets the shared env
+dev-env() {
+	local service="${1}"
+	local config="${2}"
+	local container=`_devtools-container $service`
+
+	if [[ $config != '' ]]; then
+		local cmd="grep -i $config"
+	else
+		local cmd="cat"
+	fi
+
+	if [[ $service == service ]]; then
+		local path=/var/www/service/shared/config/environment.php
+	elif [[ $service == public ]]; then
```

Comment:
@akeiper What about `admin` and `api` ?

Reply:
@ZurabWeb admin does not have shared

Reply:
@akeiper 

![image](https://user-images.githubusercontent.com/12236085/37232471-23000850-23bd-11e8-8714-61704dd383bd.png)

Reply:
that is your local configured, this reads the shared